from wave_pay.exceptions.unexpected_error import UnexpectedError


class InvalidResponseError(UnexpectedError):
    pass
